tools/pd-ctl/pdctl/command/scheduler.go

```diff
@@ -56,27 +56,43 @@ func NewPauseSchedulerCommand() *cobra.Command {
 	c := &cobra.Command{
 		Use:   "pause <scheduler> <delay>",
```

Comment:
Should we add help info about the unit of the delay time?

Reply:
rename to `delay_second`